pr-dhl-woocommerce/includes/class-pr-dhl-wc-product-paket-editor.php

```diff
@@ -0,0 +1,95 @@
+<?php
+/**
+ * Product editor handler class.
+ *
+ * @package PR_DHL_WC_Product
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+use Automattic\WooCommerce\Admin\BlockTemplates\BlockInterface;
+
+/**
+ * Product editor handler.
+ */
+if ( ! class_exists( 'PR_DHL_WC_Product_Paket_Editor' ) ) :
+
+class PR_DHL_WC_Product_Paket_Editor {
```

Comment:
I think its better to create an abstract class, for product editor and moving the compatibility declaration to it.

```suggestion
class PR_DHL_WC_Product_Editor_Paket extends PR_DHL_WC_Product_Editor {
```